packages/components/src/popover/utils.js

```diff
@@ -105,3 +105,109 @@ export const getFrameOffset = ( document ) => {
 	const iframeRect = frameElement.getBoundingClientRect();
 	return { x: iframeRect.left, y: iframeRect.top };
 };
+
+export const getReferenceOwnerDocument = ( {
+	// @ts-ignore
```

Comment:
For now, I haven't added types or JSDocs because I'm planning on rewriting this component to TypeScript relatively soon (plus, I hope we can get rid of a few permutations soon enough)